lib/offline.js

```diff
@@ -42,8 +48,9 @@ module.exports = function(config) {
     }
 
     var fileGlobs = config.fileGlobs || ['**/*'];
+    var importScripts = config.importScripts || [];
 
-    // Remove the existing service worker, if any, so sw-precache doesn't include
+    // Remove the existing service worker, if any, so the worker doesn't include
     // it in the list of files to cache.
```

Comment:
Nit: with this change, I'd also modify the sentence to read, "so the worker doesn't include *itself* in the list of files to cache."